zproject/urls.py

```diff
@@ -498,6 +498,10 @@
     url(r'^json/', include(v1_api_and_json_patterns)),
 ]
 
+# This url serves the number of online users to be set on the README.md
+# community button
+urls += url(r'^badge/users', backends.get_number_online_users, 'users')
```

Comment:
the name ('users') should be the same as the path, and is a keyword arg. So `name='backends.get_number_online_users'` in this case.

Reply:
Oh, got it.